services/press-fault-data-simulator-service/app/main.py

```
@@ -0,0 +1,35 @@
+from fastapi import FastAPI
+from contextlib import asynccontextmanager
+import os
```

Ruff (0.12.2)

`os` imported but unused

Remove unused import: `os`

(F401)